### PR TITLE
[19]: Very basic error page configuration

### PR DESCRIPTION
**Pull request by RobertSaylesAND**
[Ticket](https://github.com/isaaccomputerscience/isaac-cs-issues/issues/19)
In theory, this should cause all errors to get redirected to the generic error page defined in the front-end. Documentation isn't entirely clear as to whether this will work or if we need to define every error code/type we want to affect but I'm feeling somewhat optimistic... We might want something more fine-tuned before we call this done but it is at least a starting point.
Warning: this is currently untested, as currently we don't seem to be able to run locally with the full Jetty configuration; testing in staging will probably be required...

### CODE REVIEW

**Review by charlescolbourn — APPROVED** [review]

File: config-templates/content_indices.cs.properties
Comment (charlescolbourn) [inline config-templates/content_indices.cs.properties:1]: this probably shouldn't have been checked in?

File: web-api-local.xml
Comment (charlescolbourn) [inline web-api-local.xml:1]: is this still needed? I thought the jetty.xml was required to do this bit?

Comment (KasimAliAND) [reply web-api-local.xml:1]: Changed the local to be the same as live. Probably isn't required and should be able to remove the same from the live xml. I just wanted it to be consistent.

File: config-templates/content_indices.cs.properties
Comment (KasimAliAND) [reply config-templates/content_indices.cs.properties:1]: I have backed these changes out in my latest commit

File: web-api-local.xml
Comment (charlescolbourn) [reply web-api-local.xml:1]: Might be worth removing it from both. If we don't think it has an effect it's just noise - we can verify the effect in staging before putting live.

Comment (KasimAliAND) [reply web-api-local.xml:1]: Removed them both and tested on my local environment. Still working as intended

**Pull request merged**